docs/osf-trivia-list/dts.md

```diff
@@ -40,9 +40,9 @@ This section describes how to do it.
   successful, the message `Verification of the Dasharo DES was successful. They
```

Comment:
Currently there is no such message:

```
Dasharo DPP credentials have been saved
Checking for Dasharo firmware...
Waiting for network connection ...
Network connection have been established!
Checking if board is Dasharo compatible.
Waiting for network connection ...
Network connection have been established!
A Dasharo Firmware binary has been found for your platform!
```

Should be changed to either `A Dasharo Firmware binary has been found for your platform!` (should probably be fixed in dts-script to say that authorization is ok, somewhat related to https://github.com/Dasharo/dasharo-issues/issues/1063) or maybe say that in case of wrong credentials DTS will print error

Reply:
Added both. The problem has been solved by 9cd1e486637db4a6908efb7c509af04e860a9cb2

Reply:
I may also take care of the issue you linked in the future, if I will have time